### PR TITLE
give verificator methods its own namespace

### PR DESCRIPTION
**Pull request by microstudi**
With this PR verfication is handled in its own namespace.
The 3rd engine will provide the delegation authentication.

This also includes the delegation_verifier method automatically to the first organization (so no need to activated manually in the system for development).

### CODE REVIEW

File: lib/decidim/action_delegator/verification/engine.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+module Decidim
+  module ActionDelegator
+    module Verification
+      # This is the engine that runs on the public interface of `ActionDelegator`.
+      class Engine < ::Rails::Engine
+        isolate_namespace Decidim::ActionDelegator::Verification
+
+        paths["db/migrate"] = nil
+        paths["lib/tasks"] = nil
```
Comment (sauloperez) [inline lib/decidim/action_delegator/verification/engine.rb:11]: we should remove these two for the migrations and tasks to work, as we did in https://github.com/coopdevs/decidim-module-action_delegator/pull/2/, right?

File: lib/decidim/action_delegator/verification/engine.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+        routes do
+          # routes
+        end
+
+        def load_seed
+          # Enable the `:delegations_verifier` authorization
+          org = Decidim::Organization.first
+          org.available_authorizations << :delegations_verifier
+          org.save!
```
Comment (sauloperez) [inline lib/decidim/action_delegator/verification/engine.rb:21]: :ok_hand:

File: lib/decidim/action_delegator/verification/admin.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ActionDelegator
+    module Verification
+      # This module contains all the domain logic associated to the
+      # delegations verfier authorization admin panel.
```
Comment (sauloperez) [inline lib/decidim/action_delegator/verification/admin.rb:7]: :warning: type with `verfier`

File: lib/decidim/action_delegator/verification/engine.rb (hunk 1 above)
Comment (microstudi) [reply lib/decidim/action_delegator/verification/engine.rb:11]: I think they should be nil here, the ActionDelegator::Engine will hold those migrations isn't? we only need one engine with the migrations

**Comment by microstudi** [comment]
> why are we removing `app/controllers/decidim/action_delegator/admin/delegations_controller.rb`? that's the one we're using to manage delegations in admin. Or am I missing something?

Because verification in Decidim are handled in a particular way, there registered as part of a workflow (see `workflow.rb`). Their routes are mounted under the verification zone in the admin, so it seems natural to namespace it accordingly.
What we would should think if we want 2 admin places, one for the verifications and one for the delegations. Or integrate all together. If we go for the 2 admin menus, we would need to create a 4th engine (as we need a verfication admin module for importing the csv and such)

**Pull request merged**